### PR TITLE
Adding a Blog Documentation + Thumbnail Template

### PR DESCRIPTION
**Pull request by simoninithomas**
Hey there 👋 

I created a blog documentation with some small advices:
- How to write LaTeX.
- How to create a responsive thumbnail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# The Hugging Face Blog Repository 🤗
```
Comment (osanseviero) [inline README.md:1]: Maybe let's add one-two sentences saying this is the official repository of hf.co/blog

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## How to get a responsive thumbnail?
+1️⃣ Create a `1300x650` image 
+
+2️⃣ Use [this template](https://github.com/huggingface/blog/blob/Documentation/assets/thumbnail-template.svg) and fill the content part.
+
+
+## I want to write LaTeX
```
Comment (osanseviero) [inline README.md:8]:
```suggestion
## Using LaTeX
```

**Review by osanseviero — APPROVED** [review]
Great idea! Thanks for this!

File: README.md (hunk 1 above)
Comment (simoninithomas) [reply README.md:1]: :heavy_check_mark:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# The Hugging Face Blog Repository 🤗
+This is the official repository of the [Hugging Face Blog](hf.co/blog). 
+
+
+## How to get a responsive thumbnail?
+1️⃣ Create a `1300x650` image 
+
+2️⃣ Use [this template](https://github.com/huggingface/blog/blob/Documentation/assets/thumbnail-template.svg) and fill the content part.
```
Comment (julien-c) [inline README.md:8]: change to `main` branch for when this will get merged

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# The Hugging Face Blog Repository 🤗
+This is the official repository of the [Hugging Face Blog](hf.co/blog). 
+
+
+## How to get a responsive thumbnail?
+1️⃣ Create a `1300x650` image
```
Comment (julien-c) [inline README.md:6]: is this the correct size with the new design @gary149 @mishig25 (and with the social platforms)? (i was using 1200x648 before)

Comment (osanseviero) [reply README.md:6]: Friendly ping on this one :D

Comment (julien-c) [reply README.md:6]: unless objection i think we can merge like this

Comment (gary149) [reply README.md:6]: I see 1160x580 in the @Reeray [figma template](https://www.figma.com/file/sXrf9VtkkbWI7kCIesMkDY/HF-Blog-Template?node-id=390%3A514). But no objection to merge;

Comment (osanseviero) [reply README.md:6]: Yes, it would be nice to merge this as some docs is better than no docs :D

Comment (simoninithomas) [reply README.md:6]: Okay let's merge 😄 🚀

**Pull request merged**